site-packages/integralstor_utils/config.py

```diff
@@ -115,6 +117,18 @@ def get_minimum_user_uid_gid(type='user'):
         return id, None
 
 
+def get_normal_users():
+    try:
+        ret, err = command.get_command_output("awk -F'[/:]' '{if ($3 >= 1000 && $3 < 65534) print $1}' /etc/passwd")
```

Comment:
Yes, that is why the function was named as get_normal_users and not get_integralstor_normal_users. The differentiation between normal and system users is generally from the operating system's point of view. 

This function gets all such normal users, i.e. normal to the OS, and not the normal users that are specific to Integralstor. There must be another function which will get the intergralstor specific normal users and we don't have it yet; will put it in soon though.